packages/ogmios/ogmios-v6.1.0.yaml

```diff
@@ -22,6 +24,13 @@ installSteps:
       ports:
         - "1337"
       pullOnly: false
+outputs:
+  - name: ogmios_url
+    description: Ogmios WebSocket & HTTP server URL
+    value: 'http://localhost:1337/'
+  - name: ogmios_health_url
+    description: Ogmios helth URL
+    value: 'http://localhost:1337/health'
```

Comment:
This is the container port, but we'll want to provide the host port for the user. You should be able to get the port mapping with something like `{{ .Ports.ogmios["1337"] }}`

Reply:
This gives me an issue with bad character "["